nixos/modules/services/matrix/mautrix-facebook.nix

```diff
@@ -116,6 +122,8 @@ in {
   };
 
   config = mkIf cfg.enable {
+    users.groups.mautrix-facebook = {};
+
     users.users.mautrix-facebook = {
       group = "mautrix-facebook";
```

Comment:
How did this ever work, if the group was never added? It looks like the group has been set for a while, unless I'm missing something? (Or is it as simple as it just never worked?)

Reply:
I think it works anyways, just emits a warning. So this basically squashes the warning.

Reply:
Is your commit description incorrect, then? (When was the group `nobody`?)

Reply:
Ah it probably is. I was thinking about when the default `users.users.*.group` was `nobody`. But this isn't applicable here since we explicitly have a group.

Reply:
If the group is explicitly set but that group doesn't exist, we [default to `nobody`](https://github.com/NixOS/nixpkgs/blob/f7bb0cf1042b8565f4d8881fa2da2cedc15a979b/nixos/modules/config/update-users-groups.pl#L204). (I assume the warning you're talking about is `warning: user ‘mautrix-facebook’ has unknown group ‘mautrix-facebook’`?)